xwiki-commons-tools/xwiki-commons-tool-xar/xwiki-commons-tool-xar-plugin/src/test/java/org/xwiki/tool/xar/FormatMojoTest.java

```diff
@@ -146,7 +150,28 @@ void formatSpecialContentFailingWithXercesFromJDK8() throws Exception
         writer.setVersion("1.1");
         writer.write(domdoc);
         writer.close();
-
-        assertEquals(expectedContent, baos.toString());
+        String actual = baos.toString(Charset.forName("UTF-8"));
```

Comment:
I think what you were searching for is `java.nio.charset.StandardCharsets.UTF_8`.